cupy/padding/pad.py

```diff
@@ -1,321 +1,738 @@
+import cupy
 import numpy
-import six
+from numpy.lib.index_tricks import ndindex
```

Comment:
In this project, importing non-module symbols is prohibited. Why not writing just as `numpy.ndindex` without importing `lib.index_tricks`?